test/decl/protocol/protocols.swift

```diff
@@ -513,16 +513,15 @@ extension LetThereBeCrash {
 }
 
 protocol SomeProtocol {
- associatedtype SomeType
- func someFunction()
- func anotherFunction()
+	func someFunction()
+	func anotherFunction()
 }
 
 extension SomeProtocol {
 	
- func someFunction() {
-  let _ = #selector(anotherFunction) // expected-error {{cannot use 'anotherFunction' as a selector due to associatedtype requirement on 'SomeProtocol'}}
- }
+	func someFunction() {
+		let _ = #selector(anotherFunction) // expected-error {{cannot use 'anotherFunction' as a selector because protocol 'SomeProtocol' is not exposed to Objective C}}
```

Comment:
You can test that there are no fix-its attached to an `expected-*` diagnostic by adding `{{none}}` after the first `{{...}}`.

(Also nitpick: tabs snuck in here.)

Reply:
Fixed!